custom_components/elegoo_printer/elegoo_sdcp/models/enums.py

```
@@ -241,3 +241,15 @@ def from_int(cls, status_int: int) -> Optional["ErrorStatusReason"]:
             return cls(status_int)
         except ValueError:
             return None
+
+class PrinterType(Enum):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)